oidc-controller/api/core/config.py

```diff
@@ -69,6 +69,7 @@ class GlobalConfig(BaseSettings):
     TRACTION_CORS_URLS: str = os.environ.get("TRACTION_CORS_URLS", "")
 
     # openssl rand -hex 32
+    SIGNING_KEY_FILENAME = os.environ.get("SIGNING_KEY_FILENAME", "signing_key.pem")
```

Comment:
If we can use this to load an externally-generated `pem` file we might want to rename to `SIGNING_KEY_FILE` and accept an absolute file path (I am thinking we might mount a volume in a deployment to preserve the keys between restarts - which path is that by default?).

Reply:
yea, this who thing needs a revisit. related to my comments above. I was just trying to get the tests work